src/diffusers/pipelines/animate_diff/motion_module.py

```diff
@@ -0,0 +1,557 @@
+from dataclasses import dataclass
+from typing import List, Optional, Tuple, Union
+
+import torch
+import numpy as np
+import torch.nn.functional as F
+from torch import nn
+import torchvision
+
+from ...configuration_utils import ConfigMixin, register_to_config
+from ...models.modeling_utils import ModelMixin
+from ...utils import BaseOutput
+from ...utils.import_utils import is_xformers_available
+from ...models.attention_processor import Attention
+from ...models.attention import FeedForward
+
+from einops import rearrange, repeat
```

Comment:
```suggestion
```

No einops in `diffusers` please . We don't intend to add `einops` as a dependency